core/src/lib.rs

```diff
@@ -12,6 +12,7 @@ pub mod ledger;
 pub mod proto;
 pub mod types;
 
+pub use ethbridge_structs;
```

Comment:
we have `ethbridge_structs` re-exported under `core/src/types/ethereum_structs.rs`
```suggestion
```